### ISSUE TITLE
Others cannot join without mod [Bug] 

### ISSUE THREAD

**Issue by Theoneandonlyf** [isXander/Controlify#318]
### Current Behaviour

When I host a world with the Controlify mod enabled, others cannot join unless they also have it installed.

### Expected Behaviour

For players to join without the mod themselves.

### Screenshots

![image](https://github.com/isXander/Controlify/assets/98689506/1f8ac1e6-e6a0-44b0-81c3-97d1d7eb27d7)

### Reproduction Steps

1. Host world from single player with Controlify mod.
2. Another user attempt to join server without Controlify mod.
3. Unable to join without mod.

### Mod Version

2.0.0-beta.8 for MC 1.20.6

### Controller

issues

### Bluetooth

- [ ] Yes

### Operating System

Windows

### ARM

- [X] Yes

### Just to make sure...

- [X] I have made sure I am using the latest version of Controlify for the latest version of Minecraft.
- [X] I have made sure there are no other issues describing the same problem on the [issue tracker](https://github.com/isXander/Controlify/issues).

**Comment by isXander** [comment]
Some other mod makes this screen. Controlify does not prevent you/others from joining servers with/without Controlify.

**Comment by Theoneandonlyf** [comment]
> Some other mod makes this screen. Controlify does not prevent you/others from joining servers with/without Controlify.

False, there is only fabric api, Yet another config Lib, and Controlify on the host. The player joining has fabric api and Yet another config Lib. 

Which leaves only fabric api, which is unlikely. Or YACL.

**Comment by Kichura** [comment]
Can reproduce and this has been a problem in Blanketcon where controlify also made an appearance there.

The screenshot of the whole thing can be observed [here](https://i.imgur.com/bydTAD5.png) - Left side of both monitors show client 1 and the right side of both monitors show client 2,
But in case that's not enough - The full logs of client 1 can be found [here](https://pastebin.com/raw/GMC4gf3Z) and client 2 can be found [here](https://pastebin.com/raw/tjNcEnJ3).

@isXander therefore this issue is not invalid and should be reopened for investigation whenever possible.

**Comment by isXander** [comment]
Sorry for closing so quickly...
This looks like an issue with the UI sound that controlify registers, looks like it's complaining when the client doesn't have it. Weird.
